package.json

```diff
@@ -73,6 +74,7 @@
     "ionic-mocks-jest": "^1.3.3",
     "jest": "^23.6.0",
     "jest-preset-angular": "^6.0.1",
+    "json-schema-to-typescript": "^6.1.0",
```

Comment:
For awareness I think we may go down the path of centralising these JSON schemas and TS interfaces in another repo to be pulled in by the app/microservices. Can stay for now but we will probably remove later..